budget-binder-multiplatform-app/src/commonMain/kotlin/de/hsfl/budgetBinder/presentation/Screen.kt

```diff
@@ -13,8 +13,8 @@ sealed class Screen {
     data class CategoryEdit (val id: Int) : Screen()
     object CategoryCreate : Screen()
     object CategoryCreateOnRegister : Screen()
-    data class EntryOverview (val id: Int) : Screen()
-    data class EntryEdit (val id: Int): Screen()
+    data class EntryOverview (val id: Int, val categoryList :List<Category>) : Screen()
+    data class EntryEdit (val id: Int, val categoryList :List<Category>): Screen()
```

Comment:
I'm not really happy with giving them the category list but we would need the getCategories useCase in EntryViewModel otherwise and that seems ... weird? So I'll accept this